roxie/ccd/ccdserver.cpp

```diff
@@ -937,7 +938,7 @@ class CRoxieServerActivity : implements CInterfaceOf<IRoxieServerActivity>, impl
 
     mutable CRuntimeStatisticCollection stats;
     MapStringToMyClass<ThorSectionTimer> functionTimers;
-    ActivityTimeAccumulator totalCycles;
+    ActivityTimeAccumulator activityStats;
```

Comment:
In general it would be better to have changes that rename variables (or fix whitespace issues) in separate PRs - otherwise they are in danger of hiding the significant changes.

Reply:
The rename was in a separate commit before Jake completed the review and asked me to squash.  I think I should able to undo the squash.